### PR TITLE
CentOS Updates

### PR DESCRIPTION
**Pull request by nniehoff**
## Fixes: #166 
 

This PR updates the CentOS Install procedures:
- Updating to python38 as it is available from the default repos with CentOS/RHEL 8.  
- There are a couple of typos which are also corrected here.  
- Updates the nginx configuration instructions to be more aligned with the default RHEL/CentOS default procedures from the OS vendor
- Removes the Apache configuration all together as it was Ubuntu specific and nginx is the recommended HTTP server

### CODE REVIEW

**Comment by nniehoff** [comment]
I am marking this as draft as I have run into further issues with CentOS on certain platforms.

**Comment by jathanism** [comment]
I am okay with this. Thanks!

**Comment by nniehoff** [comment]
I think it's worth mentioning I removed the server name from the nginx https configuration and made nautobot the default http/https server.  I wanted to see what people thought of that and then probably need to do a little cleanup of the config and the docs to reflect that decision.

**Comment by glennmatthews** [comment]
@nniehoff Please work with @jathanism to resolve any merge conflicts between this PR and #189, as you're both touching the http-server.md documentation. Thank you for this PR!

File: nautobot/docs/installation/http-server.md [hunk 1]
```diff
@@ -130,39 +114,20 @@ $ sudo rm -f /etc/nginx/sites-enabled/default
...
-```
-
-### Apache
-
-[Apache](https://httpd.apache.org/docs/current/) is a tried and true secure, efficient and extensible HTTP server that
-has been around since 1995.
+On CentOS:
 
-Begin by installing Apache:
+Run the following command:
```
Comment (jathanism) [inline nautobot/docs/installation/http-server.md:119]: Can you please just add a little explanation on why this command should be ran?

**Review by jathanism — APPROVED** [review]

File: nautobot/docs/installation/http-server.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ $ sudo ln -s /etc/nginx/sites-available/nautobot.conf /etc/nginx/sites-enabled/n
 
 On CentOS:
 
-Run the following command:
+Run the following command to disable the default site that comes with the nginx package:
```
Comment (jathanism) [inline nautobot/docs/installation/http-server.md:119]:
```suggestion
Run the following command to disable the default site that comes with the `nginx` package:
```

**Pull request merged**